bndtools.core/src/bndtools/central/Central.java

```diff
@@ -655,4 +587,57 @@ public static void removeRepositoriesViewer(TreeViewer viewer) {
     public static void setRepositories(TreeViewer viewer, RefreshModel model) {
         repositoriesViewRefresher.setRepositories(viewer, model);
     }
+
+    public static void refreshRepositoriesViewers() {
+        repositoriesViewRefresher.updateViewers();
+    }
```

Comment:
Not sure if this is needed any more since the new onWorkspaceInit-callback solution